scripts/integration.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+go run main.go instances add dev
```

Comment:
oh... but that would mean you could not hit "localhost:8080"... meh, nevermind if its too complex